terraform/gcp/projects/leap.tfvars

```diff
@@ -0,0 +1,75 @@
+prefix                 = "leap"
+project_id             = "leap-pangeo"
+core_node_machine_type = "n1-highmem-4"
+
+# No need for this to be a private cluster, public ones are cheaper
+enable_private_cluster = false
+
+# Multi-tenant cluster, network policy is required to enforce separation between hubs
+enable_network_policy  = true
+
+# Some hubs want a storage bucket, so we need to have config connector enabled
```

Comment:
This comment is mismatching with the below `false` value... maybe the comment could be updated to explain why the config_connector is disabled here.

Reply:
I added a FIXME here, stating that config_connector is broken and shall be fixed with #1130